pdf-html/src/test/java/com/lowagie/text/html/simpleparser/FactoryPropertiesTest.java

```diff
@@ -0,0 +1,26 @@
+package com.lowagie.text.html.simpleparser;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
```

Comment:
Does it really need? Maybe just removing the JavaDoc is better? It is a unit test class.